x-pack/plugins/apm/server/routes/source_maps/create_apm_source_map_index_template.ts

```diff
@@ -17,10 +17,10 @@ const indexTemplate: IndicesPutIndexTemplateRequest = {
     index_patterns: [APM_SOURCE_MAP_INDEX],
     template: {
       settings: {
-        auto_expand_replicas: '0-2',
-        number_of_replicas: 0,
         number_of_shards: 1,
```

Comment:
Shouldn't `number_of_shards` be inside `index` too?

Reply:
Fixed